### PR TITLE
chore(release): bump version to 0.12.64

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,35 @@
 
+<a name="0.12.64"></a>
+## [0.12.64](https://github.com/garden-io/garden/compare/0.13.13...0.12.64) (2023-08-22)
```
Comment (shumailxyz) [inline CHANGELOG.md:3]: This seems to be off

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+
+### Bug Fixes
+
+* escape rsync special characters in filenames on windows ([#4434](https://github.com/garden-io/garden/issues/4434)) ([#4438](https://github.com/garden-io/garden/issues/4438)) ([8247044a6](https://github.com/garden-io/garden/commit/8247044a6))
+* update version hash if a file is renamed ([0c32af3da](https://github.com/garden-io/garden/commit/0c32af3da))
```
Comment (shumailxyz) [inline CHANGELOG.md:8]: Also, this was done already in one of the past releases. Seems like it's picking up more changelog than it should

File: CHANGELOG.md (hunk 1 above)
Comment (vvagaytsev) [reply CHANGELOG.md:3]: Yes, I'll remove this. We have to do the same manual removal every 0.12 release.

Comment (vvagaytsev) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (vvagaytsev) [reply CHANGELOG.md]: Yes, it does, I removed everything unrelated in a6fd638a2987c9a6c4e50d44ab2a24d4eaf1e404

**Pull request merged**